### PR TITLE
Meson

### PR DESCRIPTION
**Pull request by kif**

### CODE REVIEW

File: fabio/ext/meson.build [hunk 1]
```diff
@@ -0,0 +1,49 @@
+py = import('python').find_installation()
+dep_py = py.dependency()
```
Comment (eli-schwartz) [inline fabio/ext/meson.build:2]: You already have this defined from the top-level meson.build, so you do not need to re-import it.

File: meson.build [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+endif
+
+# https://mesonbuild.com/Python-module.html
+py_mod = import('python')
+py = py_mod.find_installation()
+py_dep = py.dependency()
+
+py.install_sources(
+  ['version.py'],
+  #rename: ['_version.py'],
```
Comment (eli-schwartz) [inline meson.build:23]: You should be able to do this with 

```meson
version_py = configure_file(input: 'version.py', output: '_version.py', copy: true)
py.install_sources(
    version_py,
    pure: false,
    subdir: 'fabio',
)
```

However, you could also move this to fabio/meson.build, use `input: '../version.py'`, and simply add `version_py` to `python_sources`.

**Review by eli-schwartz — CHANGES_REQUESTED** [review]

File: fabio/ext/meson.build (hunk 1 above)
Comment (kif) [reply fabio/ext/meson.build:2]: Thanks, I a complete newbe with meson

**Comment by kif** [comment]
Mind to use a patched version of meson-python:
https://github.com/FFY00/meson-python/issues/80

Beside this, the structure is there. The complete code is not ported to meson, but should be trivial.

Remains to be done: update all utilities around packaging like
* [ ] bootstrap
* [ ] run-tests
* [ ] debian packaging
* [ ] documentation

**Pull request merged**